conf/cassandra.yaml

```diff
@@ -203,16 +225,27 @@ permissions_validity_in_ms: 2000
 # underlying table, it may not  bring a significant reduction in the
 # latency of individual authentication attempts.
 # Defaults to 2000, set to 0 to disable credentials caching.
+# For a long-running cache using credentials_cache_active_update, consider
+# setting to something longer such as a daily validation: 86400000
 credentials_validity_in_ms: 2000
 
 # Refresh interval for credentials cache (if enabled).
 # After this interval, cache entries become eligible for refresh. Upon next
 # access, an async reload is scheduled and the old value returned until it
 # completes. If credentials_validity_in_ms is non-zero, then this must be
 # also.
+# This setting is also used to inform the interval of auto-updating if
+# using credentials_cache_active_update.
 # Defaults to the same value as credentials_validity_in_ms.
+# For a longer-running permissions cache, consider setting to update hourly (60000)
 # credentials_update_interval_in_ms: 2000
 
+# If true, cache contents are actively updated by a background task at the
+# interval set by credentials_update_interval_in_ms. If false, cache entries
+# become eligible for refresh after their update interval. Upon next access,
+# an async reload is scheduled and the old value returned until it completes.
+# credentials_cache_active_update: true
```

Comment:
For all three of these, the commented YAML value is `true`, but the default from `Config` appears to be `false`. I'd either make them both `false` or just clarify that the default is `false` in the comment here. WDYT?

Reply:
At the very least, the default value in Config.java needs to match what's in our .yaml. It strikes me that this would be something we could pretty easily lint in the future (both variable names and also consistent defaults).